File: RFID and Telegram bot/connDB.py
import MySQLdb

db = MySQLdb.connect(
    host="",  #DB的所在位址IP
    user="",  #使用者名稱
    port=3306,
    passwd="", #使用者名稱密碼
    db=""  #DB的名稱
)
cursor = db.cursor()

File: web/connDB.php
<?php
    $host = '192.168.1.66'; #資料庫的UP位址
    $user = 'root'; #使用者帳號
    $passw = '123456789';  #使用者密碼
    $dbName = 'blc';  #資料庫名稱

    $db_link=@mysqli_connect($host, $user, $passw, $dbName);
    if(!$db_link) 
        die("Could not connect : " . mysqli_error());  
    mysqli_set_charset($db_link,"utf8");

?>
